app/emoji.go

```
@@ -80,7 +89,15 @@ func (a *App) CreateEmoji(sessionUserId string, emoji *model.Emoji, multiPartIma
 }
 
 func (a *App) GetEmojiList(page, perPage int, sort string) ([]*model.Emoji, *model.AppError) {
-	return a.Srv().Store.Emoji().GetList(page*perPage, perPage, sort)
+	list, err := a.Srv().Store.Emoji().GetList(page*perPage, perPage, sort)
+	if err != nil {
+		switch {
```

Same as below. Let's just return the error directly.